src/parse.rs

```diff
@@ -459,8 +459,8 @@ fn parse(text: &str, opt: ParsingOptions) -> Result<Document, Error> {
     };
 
     // Trying to guess rough nodes and attributes amount.
-    let nodes_capacity = text.bytes().filter(|c| *c == b'<').count();
-    let attributes_capacity = text.bytes().filter(|c| *c == b'=').count();
+    let nodes_capacity = text.match_indices('<').count();
+    let attributes_capacity = text.match_indices('=').count();
```

Comment:
Curious, why `match_indices` instead of `matches`?

Reply:
No reason besides looking through the methods and not seeing `matches`. Will change.